util/masking.py

```diff
@@ -149,11 +149,22 @@ def generate(self, imageset):
         masks = []
         for index, (im, panel) in enumerate(zip(image, detector)):
 
-            # Create the basic mask from the trusted range
             if self.params.use_trusted_range:
+                trusted_mask = None
                 low, high = panel.get_trusted_range()
-                imd = im.as_double()
-                mask = (imd > low) & (imd < high)
+                trusted_mask = None
+                for image_index in imageset.indices():
```

Comment:
Could we include a comment here explaining how the trusted range is determined in people-words (this description would probably also be worth including in the commit message when the PR is squash/merged)?